xmind/tests/test_workbook_mixin_element.py

```diff
@@ -0,0 +1,107 @@
+import unittest
+from unittest.mock import patch, Mock, MagicMock
+from xmind.core.mixin import WorkbookMixinElement
+import logging
+from . import base
+
+
+class WorkbookMixinElementTest(base.Base):
+    """WorkbookMixinElement test"""
+    LOGGER = logging.getLogger('WorkbookMixinElementTest')
+
+    def setUp(self):
+        super(WorkbookMixinElementTest, self).setUp()
+
+        self._init_method = self._init_patch_with_name('_init', 'xmind.core.Element.__init__')
+
+        self._ownerWorkbook = MagicMock()
+        self._ownerWorkbook.getOwnerDocument.return_value = "owner"
+
+        self._setOwnerDocument_method = self._init_patch_with_name('_setOwnerDocument', 'xmind.core.Element.setOwnerDocument')
+
+    def test_init_with_parameters(self):
+        """Test __init__ method with parameters"""
+
+        obj = WorkbookMixinElement("test", self._ownerWorkbook)
+
+        self.assertEqual(obj._owner_workbook, self._ownerWorkbook)
+        self._init_method.assert_called_once_with("test")
+        self._ownerWorkbook.getOwnerDocument.assert_called_once()
+        self._setOwnerDocument_method.assert_called_once_with("owner")
+
+    def test_get_owner_workbook(self):
+        """Test getOwnerWorkbook method"""
+
+        obj = WorkbookMixinElement("test", self._ownerWorkbook)
+
+        self.assertEqual(obj.getOwnerWorkbook(), self._ownerWorkbook)
+
+    def test_set_owner_workbook(self):
```

Comment:
Where is test for setOwnerWorkbook method when _owner_workbook is already set?

Reply:
I'll add it